crates/iota-types/src/stardust/address.rs

```diff
@@ -24,3 +25,25 @@ pub fn stardust_to_iota_address_owner(
 ) -> anyhow::Result<Owner> {
     stardust_to_iota_address(stardust_address.into()).map(Owner::AddressOwner)
 }
+
+pub fn stardust_to_iota_address_owner_maybe_swap(
```

Comment:
Add a comment to the function here and below

Reply:
Done